### ISSUE TITLE
mpv stops streaming "ytdl videos" after a while when running in "force-window, idle" mode

### ISSUE THREAD

**Issue by torontal7** [mpv-player/mpv#13205]
### Important Information

- mpv version: 0.32.0
- Linux Distribution and Version: Knoppix 9.1 (based on Debian)
- Window Manager and version: metacity 3.38.0

If you're not using git master or the latest release, update.
Knoppix is a live system: the OS and its applications must not be updated.

### Reproduction steps

I use only live systems and prefer to use mpv "never closed" i.e. to not start for every file and URL a new mpv instance. Unfortunately it is not simple to achieve this. I created this config file, which can be used to reproduce the issue:

`[hm1]
force-seekable
no-terminal
force-window
idle
osd-font-size=40
autofit-larger=95%x95%
autofit-smaller=500x500
ytdl-format='bv[ext=webm][height<=?720][fps<=?30]+ba[ext=webm]/bv[height<=?720][fps<=?30]+ba/b[height<=?720]/b'
screenshot-template='/dev/shm/mpv-kepek/%f/%wH.%wM.%wS.%wT'
vlang=hu,ro,en

[hm2]
profile=hm1
input-ipc-server=/dev/shm/mpv-socket

[hm3]
profile=hm1
screenshot-format=webp
screenshot-webp-lossless=yes

[hm4]
profile=hm3
input-ipc-server=/dev/shm/mpv-socket`

Then I started mpv with "Applications Menu; Run" pasting into the run window:

`mpv --profile=hmNUMBER`

### Expected behavior

I hoped that mpv will work without problems, I can use this instance to play a lot of files, URLs (console; loadfile).

### Actual behavior

After a while (cannot say exact time, maybe some days) mpv stops to stream URLs that require youtube-dl/yt-dlp. I got the following error in console:

`[cplayer] Failed to recognize file format.`

With other URLs or local files there is no problem. Also, if in a terminal I run mpv with my hm profile, the YouTube video is played. If I start a new mpv instance with the above run window (mpv --profile=hmNUMBER) this new instance again works without problems for a while, then "it will be tired" and cannot play YouTube videos. So, I have to start new instances despite my clever config files...

About my youtube-dl: Of course, since youtube-dl became unusable (it is not updated) I have to use yt-dlp. Because my mpv version is as showed above I have to use the "symbolic link solution".

`$ file /usr/bin/youtube-dl
/usr/bin/youtube-dl: symbolic link to /usr/local/bin/yt-dlp`

### Log file

I cannot provide, see above my config. Otherwise, in terminal there is no problem, as I wrote.

### Other

A question regarding to mpv.conf file: How can be breaked a too long line? In my file "ytdl-format" line is more than 80 characters and I tried to break it using BACKSLASH character, and every possible solutions, but nothing worked: the line is parsed correctly only in 1 line.

Regarding to mpv-socket: If we start multiple mpv instances which use the same socket what happens? Is this OK?

**Comment by sfan5** [comment]
> Knoppix is a live system: the OS and its applications must not be updated.

mpv is an actively developed application: bug reports for old versions must not be processed.

**Comment by torontal7** [comment]
@sfan5 Why you do not specify this when creating the issue??? And why you have to provide mpv version when the old versions are prohibited???

**Comment by sfan5** [comment]
The issue template contains this sentence:

""If you're not using git master or the latest release, update.""

I suppose it could be clearer but it is meaning to imply that if your version is out-of-date we *will* tell you to update, and if you can't then in most cases the bug will just closed.

> And why you have to provide mpv version when the old versions are prohibited???

If you run mpv from git and it's maybe a few days or weeks out of date in which case it's crucial information. (Just like the _log file_ for example, which you have failed to provide).
We also sometimes have people who say they are on Debian 12 and that mpv is "the latest version", but in that case Debian is outdated and actually ships 0.35.1.

> And are you sure that the "accepted" version is not affected by this problem?

No, I am not sure.
Investigating potential issues in 4 years out of date software versions is a big waste of time none of the people usually browsing this issue tracker will undertake.

If you want to help others you are welcome to investigate this bug on mpv 0.37.0 and perhaps even provide a fix.

**Comment by torontal7** [comment]
@dokur0chan Like in Facebook. It does not matter how stupid you are and does not have any argument, you can always put a "laugh like"!

**Comment by torontal7** [comment]
@dokur0chan Yes, your mother...

**Comment by torontal7** [comment]
Though this issue was closed I generated a log-file (started new mpv instance: run window, mpv hm1 profile, log-file), maybe somebody is interested.
Link: [https://pastes.io/jqbospunvs](https://pastes.io/jqbospunvs)
First the first part of the file (output.txt; lines 1-647): starting and when everything was OK (I also tried to open a private playlist, that failed but other URLs worked).
And the last part (lines 12899-13091) when mpv "became tired", and "Failed to recognize file format". As I can see this was the problem: "No format found, try lowering probescore or forcing the format". But the format specification was the same as the beginning...
Maybe later I will check this problem with the current mpv version, now I do not have possibility.
